docs/myriad-configuration.md

```diff
@@ -55,7 +61,7 @@ rebalancer: false
 # Properties for the Node Manager process that's launched by myriad as a result of 'flex up' REST call.
 nodemanager:
   jvmMaxMemoryMB: 1024  # Xmx for NM JVM process.
-  user: hduser          # The user to run NM process as.
+  #Note both user and group must exist on all slaves.
```

Comment:
Actually just need to remove the comment.  User and group no longer are relevant.  The not deleting the comment was an oversight.